yarn.lock

```diff
@@ -1490,6 +1588,19 @@
     "@babel/plugin-transform-react-jsx-self" "^7.0.0"
     "@babel/plugin-transform-react-jsx-source" "^7.0.0"
 
+"@babel/preset-react@^7.6.0":
+  version "7.10.4"
+  resolved "http://nexus.mistraldev.com/content/groups/npmgroup/@babel/preset-react/-/preset-react-7.10.4.tgz#92e8a66d816f9911d11d4cc935be67adfc82dbcf"
```

Comment:
Looks like yarn is pulling preset-react from an internal package mirror

```
error An unexpected error occurred: "http://nexus.mistraldev.com/content/groups/npmgroup/@babel/helper-plugin-utils/-/helper-plugin-utils-7.10.4.tgz: getaddrinfo ENOTFOUND nexus.mistraldev.com".
```

Reply:
Done. Thanks for the heads up